### DISCUSSION TITLE
Reflect SSI-Wallet / Wallet kit EOL on the walt-id github home page ?

### DISCUSSION THREAD

**Discussion by shearichard-nec** [walt-id/.github#71] [Ideas / Feature Requests]
As SSI-Wallet and Wallet Kit are being EOLd perhaps the text that appears on https://github.com/walt-id could reflect that ? 

Both products are listed but there's no mention of their end of life date. 

Thanks.

**Comment by taminobaumann** [comment]
Hey @shearichard-nec, 

thanks for reaching out and the feedback. Absolutely, it should be reflected. 

We will update it soon!

Best
Tamino
